test/newMethods.js

```diff
@@ -0,0 +1,147 @@
+/* global describe,before,after,beforeEach,it */
```

Comment:
Will you please add these tests into the existing "methods" test file? You could potentially refactor the test suite into more discrete files at another time.

Reply:
Sure, I'll close this PR then, and submit a new one.